src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseOptions.java

```diff
@@ -411,6 +432,14 @@ public boolean process(String opt, List<String> args) {
                     }
                 },
 
+                new Option(resources, "--no-platform-link") {
```

Comment:
Repeating preceding comment: would `--no-platform-links` (plural) be a better name?